web/src/admin/common/ak-license-notice.ts

```diff
@@ -0,0 +1,24 @@
+import "@goauthentik/elements/Alert";
```

Comment:
**4**: Provide a notification mechanism for this particular setting.

After this, all the other changes are swapping out the ad-hoc request mechanisms to use the context instead.